src/main/views/subscription-add-list.njk

```diff
@@ -0,0 +1,102 @@
+{% extends "template.njk" %}
+{% from "./macros/common-components.njk" import goBack, submitButton %}
+{%- from "govuk/components/checkboxes/macro.njk" import govukCheckboxes -%}
+{%- from "govuk/components/input/macro.njk" import govukInput -%}
```

Comment:
if govukCheckboxes and govukInput nunjucks macro are not used we can remove them here

Reply:
removed.